yupana/processor/tests_report_processor.py

```diff
@@ -357,11 +354,14 @@ def download_side_effect():
         with patch('processor.report_processor.'
                    'ReportProcessor.transition_to_downloaded',
                    side_effect=download_side_effect):
+            self.processor = report_processor.ReportProcessor()
+            self.processor.report_or_slice = self.report_record
```

Comment:
```suggestion
            self.processor = report_processor.ReportProcessor()
            self.processor.report_or_slice = self.report_record
```
@patilsuraj767 - Is it possible ti move `self.processor.report_or_slice = self.report_record` under `download_side_effect`?

Reply:
That will not be possible because before running the side effect it first run [pre_delegate](https://github.com/quipucords/yupana/blob/2546817a1b162d122591ee04b29e27be83c8b537/yupana/processor/report_processor.py#L135) in report processor for which we need `self.processor.report_or_slice` to be set.